<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="TitleLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#00aaff;&quot;&gt;Cards Memory Game&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#00aaff;&quot;&gt;Guillaume Tritz--Guden &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#00aaff;&quot;&gt;&amp;amp; Nicola Di Pietro&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>811</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(190, 50, 55);             
    color: white;
    border: none;
    padding: 6px 12px;
    font-size: 18px;
    font-weight: bold;
    border-radius: 6px;

    /* Effet relief */
    border-bottom: 5px solid rgb(150, 35, 40);
}

QPushButton:hover {
    background-color: rgb(225, 80, 85);             
    border-bottom: 5px solid rgb(175, 60, 65);      
}

QPushButton:pressed {
    background-color: rgb(160, 40, 45);            
    border-bottom: 2px solid rgb(120, 30, 35);
    padding-top: 8px;                               
}
</string>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>471</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(130, 170, 90);          
    color: rgb(226, 234, 216);
    border: none;
    padding-top: 5px;
    font-size: 18px;
    font-weight: bold;
    border-radius: 6px;
    border-bottom: 5px solid rgb(110, 144, 76);
}

QPushButton:hover {
    background-color: rgb(160, 205, 115);          
    border-bottom: 5px solid rgb(120, 160, 90);    
}

QPushButton:pressed {
    background-color: rgb(110, 150, 75);           
    border-bottom: 2px solid rgb(95, 130, 70);
    padding-top: 7px;                              
}
</string>
    </property>
    <property name="text">
     <string>Jouer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
    <addaction name="separator"/>
    <addaction name="actionLoadGame"/>
   </widget>
   <addaction name="menufichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionLoadGame">
   <property name="text">
    <string>Charger une partie</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
